templates/services/merge/execution/besu.tmpl

```diff
@@ -0,0 +1,41 @@
+{{/* besu.tmpl */}}
+{{ define "execution" }}
+  execution:
+    image: ${EC_IMAGE_VERSION}
+    container_name: execution_client
```

Comment:
Please put the properties in the same order as in the other service templates.

Reply:
done